### ISSUE TITLE
Remove extra white lines in iOS flutter build

### ISSUE THREAD

**Issue by xster** [flutter/flutter#20447]
#20009 introduced extraneous blank lines in the console output during flutter builds on iOS.

```
Launching lib/main.dart on iPhone 8 in debug mode...
Starting Xcode build...
 ├─Assembling Flutter resources...                    2.4s

 └─Compiling, linking and signing...                  1.5s

Xcode build done.                                            4.9s
flutter: push
Syncing files to device iPhone 8...                          3.0s
```

**Comment by zoechi** [comment]
Can't reproduce
```
 (master) $ flutter build ios --no-codesign
Warning: Building for device with codesigning disabled. You will have to manually codesign before deploying to device.
Building com.example.flutter0Generic for device (ios-release)...
Running Xcode build...                                                  
 ├─Building Dart code...                                    20.8s
 ├─Generating dSYM file...                                   1.0s
 ├─Stripping debug symbols...                                0.7s
 ├─Assembling Flutter resources...                           1.4s
 └─Compiling, linking and signing...                         8.1s
Xcode build done.                                           33.3s
Built /Users/zoechi/dart/playground/flutter/flutter_0_generic/build/ios/iphoneos/Runner.app.

 (master) $ flutter doctor -v
[✓] Flutter (Channel master, v1.1.10-pre.205, on Mac OS X 10.14.3 18D42, locale en-AT)
```

**Comment by xster** [comment]
Woohoo, indeed. It must have been fixed in a recent change.

**Comment by zoechi** [comment]
Reopening as #27484 shows this with 
```
[✓] Flutter (Channel master, v1.2.1-pre.66, on Mac OS X 10.14.3 18D42, locale en-US)
```

I still can't reproduce with 
```
[✓] Flutter (Channel master, v1.2.1-pre.117, on Mac OS X 10.14.3 18D42, locale en-AT)
```

```
 (master) $ flutter run -d 787FAF06-6F62-43F7-926C-A0C10DCA1E66                                                                                                                                                                ├─Assembling Flutter resources...                           2.2s                                                                                                                                                              └─Compiling, linking and signing...                         6.4s                                                   
Xcode build done.                                           13.0s
```

**Comment by xster** [comment]
Woohoo, fixed by #28477
